src/index.css

```diff
@@ -0,0 +1,182 @@
+@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Russo+One&display=swap');
```

Comment:
Good job adding navigation for the cars, however, they are very difficult to navigate (I have 4 cars):

### In the mobile view:

- [ ] It is not very functional. I can never get to the car number 5, and also the button gets in the way:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/d47a3f84-46de-4d41-9da2-b2ed5fb0dbda)

- [ ] In "delete car" there is no navigation.  I cannot delete any car that does not appear on the screen.

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/92e732b8-efd2-4e70-84eb-6d995daba452)

Kindly fix these issues with the navigation so it becomes functional on your mobile view. 💪 

_note: this was tested multiple times on both Chrome and Firefox_